README.md

```diff
@@ -52,4 +50,53 @@ If you are facing a path too long exception when building this example project,
 
 ## License
 
-Syncfusion has no liability for any damage or consequence that may arise by using or viewing the samples. The samples are for demonstrative purposes, and if you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage that is related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.
\ No newline at end of file
+Syncfusion has no liability for any damage or consequence that may arise by using or viewing the samples. The samples are for demonstrative purposes, and if you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage that is related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.
+
+## Features and Benefits
+
+### Position
```

Comment:
Update the topics the way they appear in live
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-badge-view/assets/95272441/aeb94ae2-fdb1-48fd-977e-50b3c2a1a8b8)

Reply:
Updated